src/config/data/default.schema.json

```diff
@@ -52,6 +57,44 @@
       "env": "STATS_SUPPLY"
     }
   },
+  "redis": {
+    "url": {
+      "default": "redis://localhost",
+      "env": "REDIS_URL"
+    },
+    "cluster": {
+      "default": "",
+      "env": "REDIS_CLUSTER"
+    },
+    "graph": {
```

Comment:
Eventhough it's both redis, this is a whole separate service. Just add a `redis_graph` key.

Reply:
done